import-scripts/import_scripts/channel.py

```diff
@@ -27,18 +27,9 @@
 CURRENT_DIR = os.path.dirname(os.path.abspath(__file__))
 INDEX_SCHEMA_VERSION = os.environ.get("INDEX_SCHEMA_VERSION", 0)
 CHANNELS = {
-    "unstable": {
-        "packages": "nixpkgs/nixpkgs-20.09pre",
-        "options": "nixos/unstable/nixos-20.09pre",
-    },
```

Comment:
I don't remember if I verified with this split nixpkgs/nixos, but Hydra links point to `nixos/trunk-combined`, and there is no Darwin on the nixos jobsets.

(This is relatively minor. Probably should be broken out into its own issue.)